### ISSUE TITLE
Submit Errors

### ISSUE THREAD

**Issue by Wolf666** [jesseward/plex-lastfm-scrobbler#38]
I am running plex-lastfm-scrobbler on my Synology DS1515+ with DSM 5.2, was working pretty fine until some days ago, PMS 0.9.12.4, when I saw this problem:

2015-07-22 18:00:39,926 [6798] [plex_scrobble.lastfm _do_lastfm_post] [ERROR] Unable to submit post data http://ws.audioscrobbler.com/2.0/ - HTTP Error 408: Request Timeout

I cancelled cache and logs, restarted but still having:

2015-07-22 20:49:27,111 [15820] [main <module>] [DEBUG] config : mediaserver_log_location -> /volume1/Plex/Library/Application Support/Plex Media Server/Logs/Plex Media Server.log
2015-07-22 20:49:27,111 [15820] [main <module>] [DEBUG] config : session -> /root/.config/plex-lastfm-scrobbler/session_key
2015-07-22 20:49:27,111 [15820] [main <module>] [DEBUG] config : mediaserver_url -> http://localhost:32400
2015-07-22 20:49:27,111 [15820] [main <module>] [DEBUG] config : config file location -> /root/.config/plex-lastfm-scrobbler/plex_scrobble.conf
2015-07-22 20:49:27,112 [15820] [main <module>] [DEBUG] config : log_file -> /tmp/plex_scrobble.log
2015-07-22 20:49:27,112 [15820] [main <module>] [DEBUG] config : cache_location -> /tmp/plex_scrobble.cache
2015-07-22 20:49:27,112 [15820] [main <module>] [DEBUG] using last.fm session key=/root/.config/plex-lastfm-scrobbler/session_key , st_mtime=Sat Jul  4 11:21:17 2015
2015-07-22 20:49:27,112 [15820] [main main] [INFO] starting log monitor thread.
2015-07-22 20:49:27,113 [15820] [main cache_retry] [INFO] starting cache_retry thread.
2015-07-22 20:59:05,433 [15820] [plex_scrobble.plex_monitor parse_line] [INFO] Found played song and extracted library id '13503' from plex log 
2015-07-22 20:59:05,433 [15820] [plex_scrobble.plex_monitor fetch_metadata] [INFO] Fetching library metadata from http://localhost:32400/library/metadata/13503
2015-07-22 20:59:05,506 [15820] [plex_scrobble.lastfm scrobble] [INFO] submitting Raven - VI (The Murder Sessions) to last.fm.
2015-07-22 20:59:08,838 [15820] [plex_scrobble.lastfm _do_lastfm_post] [ERROR] Unable to submit post data http://ws.audioscrobbler.com/2.0/ - HTTP Error 408: Request Timeout
2015-07-22 20:59:08,839 [15820] [plex_scrobble.scrobble_cache add] [INFO] adding 'Raven' 'VI' (The Murder Sessions) to retry cache.
2015-07-22 20:59:12,970 [15820] [plex_scrobble.plex_monitor parse_line] [INFO] Found played song and extracted library id '13503' from plex log 
2015-07-22 20:59:12,970 [15820] [plex_scrobble.plex_monitor monitor_log] [WARNING] Dupe detection : 13503, not submitting

Any tip?

**Comment by shiitake** [comment]
This can often be related to something on the server side (i.e. connectivity issue, etc). 

Do you see this error frequently or was this a one off?

**Comment by Wolf666** [comment]
After update of my DS1515+ with DSM 5.2-5592-update 2 I started to see that error. 
It seems that randomly it triggers that connection issue. 

Previously it worked flawlessly. I don't have any connection issues and I also checked my firewall (pfSense) for any block. Everything is ok in my network.

For testing I started to use LastFM support of Plex.tv (yes, they just started to offer that!) and it did not miss a scrobble.
